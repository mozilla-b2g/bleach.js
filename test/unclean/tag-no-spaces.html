<td style="border-top:5px solid white;background-repeat:no-repeat;"background="foo"></td>
