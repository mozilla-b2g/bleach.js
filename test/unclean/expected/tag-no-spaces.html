<td style="border-top:5px solid white;"></td>
